bindings/python/benchmark.py

```diff
@@ -159,6 +167,28 @@ def to_arrow(use_large):
     find_arrow_all(c, {}, schema=schema)
 
 
+@bench("pymongoarrow-to-arrow-arr")
+def to_arrow_arrays(use_large):
```

Comment:
Since there are no list items in the "large" collection, I'm assuming this prints a line for the large docs variant with just a small time?

Reply:
I made it skip the large collection so it is more clear.